sendgrid.go

```diff
@@ -37,8 +37,22 @@ func createSendGridRequest(sgOptions sendGridOptions) rest.Request {
 	if options.Host == "" {
 		options.Host = "https://api.sendgrid.com"
 	}
+  
+  return requestNew(options)
+}
+
+// GetRequests returns a default request object with multiple ApiKeys.
+func GetRequests(keys []string, endpoint string, host string) []rest.Request {
```

Comment:
Could you please provide some documentation for this function's usage [here](https://github.com/sendgrid/sendgrid-go/tree/main/use-cases)?